microsim/quant_api.py

```diff
@@ -54,107 +59,173 @@ def read_data(cls,QUANT_DIR):
     @staticmethod
     def getProbablePrimarySchoolsByMSOAIZ(dfPrimaryPopulation,dfPrimaryZones,primary_probPij,msoa_iz,threshold):
         """
-        getProbablePrimarySchoolsByMSOAIZ
         Given an MSOA area code (England and Wales) or an Intermediate Zone (IZ) 2001 code (Scotland), return
         a list of all the surrounding primary schools whose probabilty of being visited by the MSOA_IZ is
         greater than or equal to the threshold.
         School ids are taken from the Edubase list of URN
         NOTE: code identical to the secondary school version, only with switched lookup tables
-        @param msoa_iz An MSOA code (England/Wales e.g. E02000001) or an IZ2001 code (Scotland e.g. S02000001)
-        @param threshold Probability threshold e.g. 0.5 means return all possible schools with probability>=0.5
-        @returns a list of probabilities in the same order as the venues
-        """
+
+        :param dfPrimaryPopulation: # needs clarification
```

Comment:
@nickmalleson be grateful if you could review some of these docstrings as there are parameters I'm unable to properly document (I don't know what they are!)